indexer/indexer_tests/feature_xml_test.cpp

```diff
@@ -0,0 +1,111 @@
+#include "testing/testing.hpp"
+
+#include "indexer/classificator_loader.hpp"
+#include "indexer/feature.hpp"
+
+#include "base/string_utils.hpp"
+
+#include "std/sstream.hpp"
+
+#include <boost/algorithm/string/join.hpp>
+
+struct TestSetUp
+{
+  TestSetUp()
+  {
+    classificator::Load();
+  }
+};
+
+static TestSetUp testSetUp;
```

Comment:
Get rid of static keyword, move TestSetUp and testSetUp to the anonymous namespace, rename testSetUp to g_testSetUp.